xfail ch1n3p04 and ch2p3p08 tests which failed with libpng 1.6.47
Upstream issue: https://github.com/randy408/libspng/issues/276

diff -ruN a/tests/images/meson.build b/tests/images/meson.build
--- a/tests/images/meson.build	2023-05-08 13:45:20.000000000 +0000
+++ b/tests/images/meson.build	2025-05-25 18:03:53.521696834 +0000
@@ -45,8 +45,8 @@
 test('cdhn2c08', test_exe, args: files('cdhn2c08.png'))
 test('cdsn2c08', test_exe, args: files('cdsn2c08.png'))
 test('cdun2c08', test_exe, args: files('cdun2c08.png'))
-test('ch1n3p04', test_exe, args: files('ch1n3p04.png'))
-test('ch2n3p08', test_exe, args: files('ch2n3p08.png'))
+test('ch1n3p04', test_exe, args: files('ch1n3p04.png'), should_fail: true)
+test('ch2n3p08', test_exe, args: files('ch2n3p08.png'), should_fail: true)
 test('cm0n0g04', test_exe, args: files('cm0n0g04.png'))
 test('cm7n0g04', test_exe, args: files('cm7n0g04.png'))
 test('cm9n0g04', test_exe, args: files('cm9n0g04.png'))
